scripts/jest-ui.js

```diff
@@ -16,6 +16,7 @@ const glob = require('glob');
 config.output.publicPath = '/lib-test/';
 config.output.path = path.resolve(__dirname, '../lib-test');
 config.output.filename = '[name].js';
+config.externals = {};
```

Comment:
Can you explain more what is this line for?

Reply:
During test we still use `webpack.config.lib.js` but since I have added `externals` there during test build webpack again excludes those modules out which will lead to tests failing because it is not able to find some modules. This is to make sure the test build is till the entire build without any excluded modules.